src/st/np3/3246C.c

```diff
@@ -219,8 +320,10 @@ void func_801B3704(Entity* self, s16 primIndex) {
     }
 }
 
+// DECOMP_ME_WIP https://decomp.me/scratch/86Cmm
```

Comment:
I didn't know. How does that work?

Reply:
The PSX version of the script still parses these. https://github.com/Xeeynamo/sotn-decomp/blob/9b9b3f649f86f635ac19778ee75378aca94a3583/tools/function_finder/function_finder_psx.py#L96

However the format of this comment is incorrect, it needs to be:
```
// DECOMP_ME_WIP function_name decomp_me_link
```

The way it works is that the script either uses the link provided in the source, or it uses the highest scoring scratch on decomp.me. DECOMP_ME_WIP is from the original version of the script that didn't have scraping, so it was the only way to save a WIP. Since adding the scraping, I don't think it's super useful anymore but I haven't removed it.

Reply:
So i should leave the comment with that new format?

Reply:
Your scratches are already being saved in the current document:
<img width="990" alt="image" src="https://github.com/Xeeynamo/sotn-decomp/assets/122322823/31e9802f-8b0c-4b10-9d38-4b15baca9cea">
Do you want to override it? That would be the purpose of DECOMP_ME_WIP now.

Reply:
I just think it's convenient to have the scratches there when you're browsing the code and looking for functions to decompile, that way you don't need to search for the function anywhere, but if we are not allowing any more DECOMP_ME_WIP comments i'll respect that.

Reply:
I don't really have a strong opinion about having them or not, but if they are added it needs to be in the right format or it breaks the CI.

Reply:
You can just click on the family tab after opening the link and you'll find the most updated one, I still think having the link in the code at the reach of a single click is useful.